1_prep/src/build_model_config.R

```diff
@@ -25,18 +28,19 @@ build_model_config <- function(meteo_feathers, lake_cell_tile_xwalk, gcm_names,
     cell_no = as.numeric(str_extract(tools::file_path_sans_ext(meteo_fl), paste0(cell_no_list, "$")))
```

Comment:
Should this not be updated to make it clear if either the spatial or data cell is to be used? Then L42 in the join would join by `data_cell_no` and not need the distinction between left and right.

Reply:
I was debating this. When we're working with the GCM data, we always use just `cell_no` (see [here](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/7_drivers_munge/src/GCM_driver_utils.R#L505) and [here](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/_targets.R#L219-L242)). And there is no `spatial_cell_no` vs `data_cell_no` when it comes to the GCM data itself. For that reason I wasn't sure if it would be confusing here to refer to the cell_no being pulled from the meteo file names as `data_cell_no`. But I can see that it could also be confusing to _not_ use `data_cell_no` here. And we are after all, pulling the `cell_no` that pertains to the meteo *data*. So I'll make that switch.